src/main.js

```diff
@@ -1,9 +1,10 @@
-import { createApp } from 'vue';
-import App from './App.vue';
+import { createApp } from "vue";
+import App from "./App.vue";
 import router from "./router/index";
 
-const app = createApp(App)
+const app = createApp(App);
 // Plugins
-app.use(router)
+app.use(router);
+
 // * Adds the app to the div with the id 'app'
-app.mount('#app')
+app.mount("#app");
```

Comment:
Was this an issue before? This seems like something that shouldn't happen in this PR

Reply:
This wasn't an issue, I think when I tried using a CSS framework, there needed to be some things added to `App.vue` (which are now deleted of course) so when I saved the file my formatter took over (e.g. `"#app"` instead of `'#app'`)